stroeer/core/v1/core_curation_service.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+/**
+ * @FileArticle ⚙︎ CurationService
+ */
+
+package stroeer.core.v1;
+
+import "stroeer/core/v1/article.proto";
+
+option go_package = "github.com/stroeer/go-tapir/core/v1;core";
+option java_multiple_files = true;
+option java_package = "de.stroeer.core.v1";
+
+/**
+* ```protobuf
+* service ArticleService {
+*   // resolve a curation and return the `articles` within this curation
+*   rpc GetCuration(GetCurationRequest) returns (GetCurationResponse) {}
+*   //
+*   rpc BatchGetCuration(BatchGetCurationRequest) returns (BatchGetCurationResponse) {}
+* }
+* ```
+*/
+service CurationService {
+  rpc GetCuration(GetCurationRequest) returns (GetCurationResponse) {}
+  rpc BatchGetCuration(BatchGetCurationRequest) returns (BatchGetCurationResponse) {}
+}
+
+/**
+* # `⚙︎ GetCuration`
+* ```protobuf
+*   rpc GetCuration(GetCurationRequest) returns (GetCurationResponse) {}
+* ```
+*
+* Fetch a curation by its id and return the [`repeated stroeer.core.v1.Article`](Article.html) this
+* curation contains. The response may be empty in case the curation does not contain any items.
+*
+* a `NOT_FOUND` status code will indicate the curation `id` does not exist.
+*/
+message GetCurationRequest {
+  int64 id = 1;
+}
+
+message GetCurationResponse {
+  repeated stroeer.core.v1.Article articles = 1;
+}
+
+/**
+* # `⚙︎ BatchGetCuration`
+* ```protobuf
+*   rpc BatchGetCuration(BatchGetCurationRequest) returns (BatchGetCurationResponse) {}
+* ```
+*
+* Fetch multiple curations by their id and return the [`repeated stroeer.core.v1.Article`](Article.html) those
```

Comment:
Do we apply publication checks/filtering of the articles on the server side or leave that to the client? We should document this decisision.

Reply:
since this service does not reside in `core` any more, I'd say its a production API which only delivers valid stuff.

A concept for _Preview_-like APIs could be beneficial, e.g.:

-  _Contentful CMS_ provides you with different keys for _Prod_ and _Preview_
- we add a field/switch to each gRPC-service that alters the _Mode_